transform.py

```diff
@@ -39,27 +42,41 @@ def remove(*args, **kwargs):
 
 remove('header')
 remove('footer')
-remove('div', attrs={'class': 'devsite-nav-responsive-sidebar-panel'})
```

Comment:
Why are these lines removed?

Reply:
I choose to only remove header, footer and left side bar.
I keep the right navigation bar for quick navigation inside page. (It is removed because it's <nav>)